src/core/QCheck.mli

```diff
@@ -334,7 +334,8 @@ module Gen : sig
   val array_repeat : int -> 'a t -> 'a array t
   (** [array_repeat i g] builds an array generator from exactly [i] elements generated by [g]. *)
 
-  val opt : 'a t -> 'a option t (** An option generator. *)
+  val opt : ?ratio:float -> 'a t -> 'a option t
+  (**  An option generator, with optional ratio of Some. *)
```

Comment:
```suggestion
  (**  An option generator, with optional ratio.
      @param ratio a float between [0.] and [1.] indicating the probability of a sample to be [Some _]
      rather than [None]. This parameter is @since NEXT_RELEASE .
  *)
```